src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoCalendarWeekRule.cs

```diff
@@ -11,64 +12,35 @@ namespace System.Globalization.Tests
 {
     public class DateTimeFormatInfoCalendarWeekRule
     {
-        // TestGetter: Call CalendarWeekRule getter method should return correct value for InvariantInfo
-        [Fact]
-        public void TestGetter()
-        {
-            VerificationHelper(DateTimeFormatInfo.InvariantInfo,
-                    CalendarWeekRule.FirstDay,
-                    false);
-        }
-
-        // TestSetter: Call CalendarWeekRule setter method should return correct value
-        [Fact]
-        public void TestSetter()
+        public static IEnumerable<object[]> CalendarWeekRule_TestData()
```

Comment:
@ellismg I'm not sure that would cleanly work here since the br-FR case doesn't follow the template for the second element.